modules/hosted-clusters-openstack-addl-ports-creating.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-openstack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hosted-clusters-openstack-addl-ports_{context}"]
+= Creating additional ports for node pools
+
+You can configure additional ports for node pools for hosted clusters that run on {rh-openstack-first}.
+
+.Prerequisites
+
+* You created a hosted cluster.
+* You have access to the management cluster.
+* The `hcp` CLI is installed.
+* Additional networks are created in {rh-openstack}.
```

Comment:
What's the difference between `{rh-openstack}` and `{rh-openstack-first}`?

Reply:
One renders as `RHOSP` at the moment, the other `Red Hat OpenStack Platform (RHOSP)`. 

Depending on what precisely is officially supported, I might have to find a way to combine those plus RHOSO (or swap with that entirely).